androidsdk/build.gradle

```diff
@@ -1,12 +1,12 @@
 apply plugin: 'com.android.library'
 
 android {
-    compileSdkVersion 23
-    buildToolsVersion '25.0.0'
+    compileSdkVersion 26
+    buildToolsVersion '27.0.3'
 
     defaultConfig {
-        minSdkVersion 14
```

Comment:
This is required to `support` libraries, right? Might wanna mention this in the mobile channel in Slack. Changing the minimum version means we're going to lose support for older devices...

Reply:
@diegoserranoa I gave a thought on that but found that we already are mentioning to support for `minSdkVersion 16`. If we see here https://github.com/Wootric/WootricSDK-Android/blob/master/README.md under `Requirements` section.

Reply:
Ah you're right. That probably was an error on the docs side. But still, if we had `minSdkVersion 14` it means that we still supported devices with version 14 & 15. I would still mention this in the channel...